### PR TITLE
[Console][MyAccount][Feature]App Switch Header buttons

### PR DESCRIPTION
**Pull request by Thumimku**
## Purpose
$Sub

### CODE REVIEW

File: apps/myaccount/src/components/shared/app-switch.tsx [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+     /**
+     * Card Background color.
+     */
+    background?: "transparent" | "default";
+    /**
+     * If a bottom margin should be added.
+     */
+    bottomMargin?: boolean;
+}
+
```
Comment (brionmario) [inline apps/myaccount/src/components/shared/app-switch.tsx:42]:
```suggestion
```

Comment (Thumimku) [reply apps/myaccount/src/components/shared/app-switch.tsx:42]: fixed in c506f2f

File: modules/react-components/src/components/card/app-switch-card.tsx [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+  * Default props for the app switch card component.
+  */
+AppSwitchCard.defaultProps = {
+     background: "default",
+     bottomMargin: true,
+     "data-testid": "app-switch-card",
+     imageSize: "mini",
+     onClick: () => null
+ };
+
```
Comment (brionmario) [inline modules/react-components/src/components/card/app-switch-card.tsx:152]:
```suggestion
```

**Comment by brionmario** [comment]
Shall we have a screenshot of the implementation 🙂

File: apps/myaccount/src/components/shared/app-switch.tsx [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+
+/**
+ * Appswitch component.
+ *
+ * @param {React.PropsWithChildren<AvatarProps>} props - Props passed in to the Avatar component.
+ * @return {JSX.Element}
+ */
+export const AppSwitch: React.FunctionComponent<AppSwitchProps> = (props: AppSwitchProps): JSX.Element => {
+    const {
```
Comment (brionmario) [inline apps/myaccount/src/components/shared/app-switch.tsx:50]:
```suggestion

    const {
```

File: apps/myaccount/src/components/shared/app-switch.tsx [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+}
+
+
+/**
+ * Appswitch component.
+ *
+ * @param {React.PropsWithChildren<AvatarProps>} props - Props passed in to the Avatar component.
+ * @return {JSX.Element}
+ */
+export const AppSwitch: React.FunctionComponent<AppSwitchProps> = (props: AppSwitchProps): JSX.Element => {
```
Comment (brionmario) [inline apps/myaccount/src/components/shared/app-switch.tsx:49]:
```suggestion
export const AppSwitch: React.FunctionComponent<AppSwitchProps> = (props: AppSwitchProps): ReactElement => {
```

File: apps/myaccount/src/components/shared/app-switch.tsx [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+ * @param { React.SyntheticEvent<HTMLElement> } e - Click event.
+ */
+const handleUserDropdownClick = (e: SyntheticEvent<HTMLElement>) => {
+    e.stopPropagation();
+};
+
+    return (
+        <>
+            {
+                showAppSwitchButton && (AppConstants.getTenant() === AppConstants.getSuperTenant()) &&
```
Comment (brionmario) [inline apps/myaccount/src/components/shared/app-switch.tsx:72]:
```suggestion
                showAppSwitchButton && (AppConstants.getTenant() === AppConstants.getSuperTenant()) && (
                    <Dropdown
                        item
                        ...
                )    
```

File: apps/myaccount/src/components/shared/app-switch.tsx [hunk 6]
```diff
@@ -0,0 +1,135 @@
...
+/**
+ * Default prop types for the App switch component.
+ * See type definitions in {@link AppSwitchProps}
+ */
+AppSwitch.defaultProps = {
+    background: "transparent",
+    bottomMargin: true,
+    ["data-testid"]: "app-switch"
+};
+
```
Comment (brionmario) [inline apps/myaccount/src/components/shared/app-switch.tsx:134]:
```suggestion
```

File: apps/myaccount/src/components/shared/header.tsx [hunk 7]
```diff
@@ -207,6 +208,17 @@ export const Header: FunctionComponent<HeaderPropsInterface> = (
                     } }
                 />
             ) }
+            extensions={ [
+                {
+                    component: (
+                        <AppSwitch
+                            bottomMargin = { true }
```
Comment (brionmario) [inline apps/myaccount/src/components/shared/header.tsx:215]: Fix other places as well.
```suggestion
                            bottomMargin={ true }
```

**Comment by Thumimku** [comment]
> Shall we have a screenshot of the implementation slightly_smiling_face

Screen cast is added in the sibling PR.. can be find via reference

**Review by brionmario — APPROVED** [review]

**Pull request merged**